docs/developer/addon-development.md

```diff
@@ -0,0 +1,62 @@
+---
+id: addon-development-guide
+title: "Addon Development Guide"
+sidebar_position: 1
+sidebar_label: Addon Development Guide
+keywords:
+  - Harvester
+  - harvester
+  - Addon Development
+Description: How to write your own Harvester Addon
```

Comment:
"addon" is not a word. It is "Add-on" and I'm working on a PR to actively reflect that: https://github.com/harvester/dashboard/pull/1110 Please adjust accordingly